src/components/learner-credit-management/cards/CourseCard.jsx

```diff
@@ -3,95 +3,144 @@
-      tabIndex="0"
+      isClickable
+      orientation={isSmall ? 'vertical' : 'horizontal'}
     >
       <Card.ImageCap
-        src={cardImageUrl || cardFallbackImg}
-        fallbackSrc={cardFallbackImg}
-        logoSrc={partners[0]?.logo_image_url}
+        src={imageSrc}
+        logoSrc={logoSrc}
         srcAlt={altText}
         logoAlt={partners[0]?.name}
       />
-      <div className="card-container">
-        <div className="section-1">
-          <p className="mb-1 lead font-weight-bold">{title}</p>
-          <p>{partners[0]?.name}</p>
-          {courseType === EXEC_COURSE_TYPE && (
-            <Badge variant="light" className="mb-4">
-              Executive Education
-            </Badge>
+      <Card.Body>
+        <Card.Header
+          title={title}
+          className="mb-0 mt-0"
+          subtitle={partners[0]?.name}
+          actions={(
+            <Stack gap={1} className="text-right">
+              <p className="h4 mt-2.5 mb-0">{priceText}</p>
+              <span className="micro">{PRICE.subText}</span>
+            </Stack>
           )}
-          {courseType !== EXEC_COURSE_TYPE && (
-            <p className="spacer" />
-          )}
-          <p className={`small ${courseType !== EXEC_COURSE_TYPE ? 'mt-5 mb-0' : ''}`}>
-            Starts {formatDate(normalizedMetadata?.start_date)} •
-            Learner must register by {formatDate(normalizedMetadata?.enroll_by_date)}
-          </p>
-        </div>
-        <Card.Section className="section-2">
-          <p className="lead font-weight-bold mb-0">{priceText}</p>
-          <p className="micro mb-5.5">Per learner price</p>
-          <Card.Footer orientation="horizontal" className="footer">
-            <Button as={Hyperlink} destination="https://edx.org" target="_blank">View course</Button>
-
-            <Button>Assign</Button>
-          </Card.Footer>
+        />
+        <Card.Section>
+          <Badge variant="light" className="ml-0">
+            {isExecEd ? BADGE.execEd : BADGE.course}
+          </Badge>
         </Card.Section>
-      </div>
+        <Card.Footer
+          orientation={isExtraSmall ? 'horizontal' : 'vertical'}
+          textElement={isExecEd ? execEdEnrollmentInfo : courseEnrollmentInfo}
+        >
+          <Button
+            // TODO: Implementation to follow in ENT-7594
+            as={Hyperlink}
+            destination="https://enterprise.stage.edx.org"
+            target="_blank"
+            variant="outline-primary"
+          >
+            {BUTTON_ACTION.viewCourse}
+          </Button>
+          <Button>{BUTTON_ACTION.assign}</Button>
+        </Card.Footer>
+      </Card.Body>
     </Card>
   );
 };
 
-CourseCard.defaultProps = {
-  onClick: () => {},
-};
-
 CourseCard.propTypes = {
-  onClick: PropTypes.func,
+  learningType: PropTypes.string.isRequired,
   original: PropTypes.shape({
-    title: PropTypes.string,
-    cardImageUrl: PropTypes.string,
+    availability: PropTypes.string,
```

Comment:
I'm seeing a prop type warning for `original.availability`. Expects a string here but, in practice from Algolia's perspective, it is an array: 

<img width="867" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/a9c5bf84-68c6-4a9b-8547-64fd47eed013">

Reply:
Fixed this prop type.